rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/RSocketError.kt

```diff
@@ -16,30 +16,55 @@
 
 package io.rsocket.kotlin
 
-sealed class RSocketError(internal val errorCode: Int, message: String) : Throwable(message) {
+public sealed class RSocketError(internal val errorCode: Int, message: String) : Throwable(message) {
```

Comment:
Public is default, why are these all needed?

Reply:
explicit api mode require to specify `public` modifier, to make sure, that all `public` apis are explicit